readme.md

```diff
@@ -56,10 +56,17 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
 - [7-Zip](http://www.7-zip.org/) (for deployment only)
 - [NSIS](http://nsis.sourceforge.net/) (for deployment only)
 
-### Mac:
+### macOS:
+
+- Xcode 8
+
+The program can also be built as a command line program using CMake. This type of build requires:
```

Comment:
That is a good example of how "Command Line" is used in the macOS space. The only issue with using that verbatim is that OpenRCT2 can't really be described as a "Tool". This is why I kept the descriptor "Command Line", but changed the last part. I could try to reword it to use "Command Line" as a noun (ex. "OpenRCT2 can be built as a binary that can be run from the command line..."), but it could make the phrasing a bit awkward. I can also change "Program" to "Application" or something else.